doc/util.md

```diff
@@ -170,6 +170,10 @@ Methods:
  - `darken`: Darken the shade of the color by an amount between 0 to 1.
  - `brighten`: Brighten the shade of the color by an amount between 0 to 1.
 
+Factories:
+- `fromRGBHexString`: Parses an RGB color from a valid hex string (e.g. #1C1C1C).
```

Comment:
```suggestion
- `fromRGBHexString`: Parses a RGB color from a valid hex string (e.g. #1C1C1C).
```

Reply:
Since RGB is an initialism, not an acronym, I believe that is correct, because R is pronounced with a vowel.